src/sys/windows/stdnet/stream.rs

```diff
@@ -0,0 +1,226 @@
+use std::io::{self, IoSlice, IoSliceMut};
```

Comment:
Same here as in the other file with regards to:
 * ordering of `impl` blocks
 * `pub(crate)`.